cmd/clusterctl/client/config/providers_client.go

```diff
@@ -113,7 +113,6 @@ func (p *providersClient) defaults() []Provider {
 	// out-of-box experience. This is an opt-in feature; other providers can be added by using the clusterctl configuration file.
 
 	// if you are a developer of a SIG-cluster-lifecycle project, you can send a PR to extend the following list.
-
```

Comment:
Please rollback unrelated formatting changes like this

Reply:
Thanks you, it is done